doc/guides/tfm/build.rst

```diff
@@ -118,3 +118,42 @@ following CMake snippet in your CMakeLists.txt file.
    The ``TFM_CMAKE_OPTIONS`` is a list so it is possible to append multiple
    options. Also CMake generator expressions are supported, such as
    ``$<1:-DFOO=bar>``
+
+Footprint and Memory Usage
+**************************
+
+The build system offers targets to view and analyse RAM and ROM usage in generated images.
+The tools run on the final images and give information about size of symbols and code being used in both RAM and ROM.
+For more information on these tools look here: :ref:`footprint_tools`
+
+Use the ``tfm_ram_report`` to get the RAM report for the secure firmware (tfm_s).
+
+.. zephyr-app-commands::
+    :tool: all
+    :app: samples/hello_world
+    :board: mps2_an521_ns
+    :goals: tfm_ram_report
+
+Use the ``tfm_rom_report`` to get the ROM report for the secure firmware (tfm_s).
+
+.. zephyr-app-commands::
+    :tool: all
+    :app: samples/hello_world
+    :board: mps2_an521_ns
+    :goals: tfm_rom_report
+
+Use the ``bl2_ram_report`` to get the RAM report for mcuboot, if enabled.
```

Comment:
just to be very precise that this is only the MCUboot built as part of TF-M.
```suggestion
Use the ``bl2_ram_report`` to get the RAM report for TF-M enabled MCUboot.
```

Reply:
The " if enabled " part is just to say that this command is not available when TFM_BL2=n.

Do you wish to separate BL2 from MCUboot in the terminology?
I was aligning with the following text:
```
The TF-M build system creates the following executable files:
* tfm_s - the secure firmware
* tfm_ns - a nonsecure app which is discarded in favor of the Zephyr app
* bl2 - mcuboot, if enabled
```

Should we update to use TF-M MCUboot (or TF-M enabled MCUboot)?

Reply:
That's a fair point ... we should perhaps update this reference as well to be more precise and state: `bl2 - TF-M MCUBoot, if enabled`.

Reply:
@microbuilder Thanks for the feedback, updated accordingly. Also changed to `TF-M secure firmware`